src/components/Course/CourseProgramLessons.tsx

```diff
@@ -56,6 +63,12 @@ export const CourseProgramLessons: React.FC<{ program: API.CourseProgram }> = ({
     );
   }, [lessonId, topicId, program, push]);
 
+  useEffect(() => {
+    topic.can_skip
+      ? setIsDisabledNextTopicButton(false)
```

Comment:
what if 
- lesson is already finished 
- or this is skippable topic (is there any, like richtex?